lib/binaries/chrome_driver.ts

```diff
@@ -30,7 +30,13 @@ export class ChromeDriver extends Binary {
 
   suffix(ostype: string, arch: string): string {
     if (ostype === 'Darwin') {
-      return 'mac32' + this.suffixDefault;
+      // after chromedriver version 2.23, the name of the binary changed
+      // They no longer provide a 32 bit binary
+      if (parseFloat(this.version()) >= 2.23) {
```

Comment:
> Do you have any suggestions? `this.version()` is a string.

You got me, I have no idea how to do this :). If you did it simply with strings, you would avoid the float inaccuracies, but that is the smaller part of the problem. You could use [`semver`](https://www.npmjs.com/package/semver) but who wants to add a dependency just for this. (Seems like it is part of NPM so it would not be an extra download, but still.)

If there are 64-bit binaries for earlier releases, you could just use `mac64` unconditionally.

> One suggestion I got from a colleague (@tomneyland) where we split on the . and check the major and minor version as ints. Maybe not the most elegant thing, but it seems pragmatic. Do you agree?

Should work well too!

(Don't take me too seriously, I am not part of the project, just happened to spot this. Whatever is fine by me.)